llvm/lib/CodeGen/AsmPrinter/AccelTable.cpp

```diff
@@ -358,8 +359,10 @@ void AppleAccelTableWriter::emit() const {
 }
 
 DWARF5AccelTableData::DWARF5AccelTableData(const DIE &Die,
-                                           const DwarfCompileUnit &CU)
-    : OffsetVal(&Die), DieTag(Die.getTag()), UnitID(CU.getUniqueID()) {}
+                                           const DwarfUnit &Unit,
+                                           const bool IsTU)
+    : OffsetVal(&Die), DieTag(Die.getTag()), UnitID(Unit.getUniqueID()),
+      IsTU(IsTU) {}
```

Comment:
Sorry, not sure I follow.

Reply:
Ah, sorry, used the wrong term - /delegating/ ctor was what I was thinking of. This ctor could call the other "init all the members" ctor, rather than initing them directly itself. It's much of a muchness though, since they don't do anything als that interesting anyway.

Reply:
I don't think it can. Unless I am missing something. When this constructor is called the DIE offset is not known yet. I guess after the call we can over-write OffsetVal with Die again?